### PR TITLE
Closes #2744: Centers titleView and urlView when both are displayed

### LINKED ISSUE #2744: TitleView is not properly centered anymore

### PR DESCRIPTION
**Pull request by sblatz**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

**Comment by sblatz** [comment]
This accidentally slipped through when making [quick changes to the PR](https://github.com/mozilla-mobile/android-components/pull/2625#discussion_r274135554). Cutting the `height` in half did not properly center the views.

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt [hunk 1]
```diff
@@ -349,8 +349,8 @@ internal class DisplayToolbar(
 
         if (titleView.isVisible()) {
             // With a title view, the url and title split the rest of the space vertically
-            titleView.measure(urlWidthSpec, halfFixedHeightSpec)
-            urlView.measure(urlWidthSpec, halfFixedHeightSpec)
+            titleView.measure(urlWidthSpec, thirdFixedHeightSpec)
+            urlView.measure(urlWidthSpec, thirdFixedHeightSpec)
```
Comment (pocmo) [inline components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: Why `/ 3` though? 🤔

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt [hunk 2]
```diff
@@ -500,6 +500,7 @@ internal class DisplayToolbar(
     }
 
     companion object {
+        internal const val MEASURED_HEIGHT_THIRD_DENOMINATOR = 3
         internal const val MEASURED_HEIGHT_DENOMINATOR = 2
```
Comment (pocmo) [inline components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: Is `MEASURED_HEIGHT_DENOMINATOR` now unused?

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt (hunk 1 above)
Comment (sblatz) [reply components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: DM'd you about this 😄

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt (hunk 2 above)
Comment (sblatz) [reply components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: No, it's used to divide the `totalPadding` in half.

**Comment by sblatz** [comment]
@pocmo, I added a better comment explaining why we divide by 3, and a test to assert that we do!

**Pull request merged**